swig/Makefile

```diff
@@ -20,7 +20,7 @@ CVODES=fcvodes_mod
 IDA=fida_mod
 IDAS=fidas_mod
 KINSOL=fkinsol_mod
-GENERIC=fsundials_core_mod
+CORE=fsundials_core_mod
```

Comment:
In most of the `src/X/X/fmod_int32/CMakeLists.txt` and `src/X/X/fmod_int64/CMakeLists.txt` files you have the line
```
    PUBLIC sundials_fcore_mod
```

Should all of those instead be
```
    PUBLIC fsundials_core_mod
```
?

Reply:
No. The latter is the name of the module but  the former is the name of the CMake target / library (confusing - yes).